v1.1.3.md

```diff
@@ -0,0 +1,61 @@
+# Harvester v1.1.3 Release Note
+
+This release follows Harvester v1.1.2. The documentation is available at https://docs.harvesterhci.io/v1.1.
+
+## Downloads
+
+### ISO
+
+:cd: [harvester-v1.1.3-amd64.iso](https://releases.rancher.com/harvester/v1.2.1/harvester-v1.2.1-amd64.iso)
+
+### Additional Artifacts
+
+:file_folder: https://releases.rancher.com/harvester/v1.2.1/harvester-v1.2.1-vmlinuz-amd64
+:file_folder: https://releases.rancher.com/harvester/v1.2.1/harvester-v1.2.1-initrd-amd64
+:file_folder: https://releases.rancher.com/harvester/v1.2.1/harvester-v1.2.1-rootfs-amd64.squashfs
+:file_folder: https://releases.rancher.com/harvester/v1.2.1/harvester-v1.2.1-amd64.raw.zst
+:white_check_mark: https://releases.rancher.com/harvester/v1.2.1/harvester-v1.2.1-amd64.sha512
+:memo: https://releases.rancher.com/harvester/v1.2.1/version.yaml
```

Comment:
@markhillgit May you update the ISO file and Additonal Artifacts when ready? Thanks!